infrastructure/main.tf

```diff
@@ -39,3 +39,16 @@ data "azurerm_key_vault_secret" "BLOB_URL_SUFFIX" {
   name = "BLOB-URL-SUFFIX"
   key_vault_id = "${data.azurerm_key_vault.rd_key_vault.id}"
 }
+
+
+
+resource "azurerm_resource_group" "rg" {
+  name = "${var.product}-${var.component}-${var.env}"
+  location = "${var.location}"
+  tags {
+    "Deployment Environment" = "${var.env}"
+    "Team Name" = "${var.team_name}"
+    "lastUpdated" = "${timestamp()}"
+  }
+}
+
```

Comment:
why is this added again? This was removed in last commit by me and should not be added back